packages/plugin-node-surrounding-code/test/surrounding-code.test.js

```diff
@@ -207,4 +207,32 @@ describe('plugin: node surrounding code', () => {
       }
     ]))
   })
+
+  it('truncates lines to 150 characters', done => {
```

Comment:
should this say 200 characters?

Reply:
yes – or better still only have the constant in the assertion and make the scenario description generic